android/src/main/java/com/onegini/mobile/sdk/reactnative/handlers/registration/RegistrationRequestHandler.kt

```diff
@@ -0,0 +1,34 @@
+package com.onegini.mobile.sdk.reactnative.handlers.registration
+
+import com.onegini.mobile.sdk.android.handlers.request.OneginiBrowserRegistrationRequestHandler
+import com.onegini.mobile.sdk.android.handlers.request.callback.OneginiBrowserRegistrationCallback
+import android.net.Uri
+
+class RegistrationRequestHandler : OneginiBrowserRegistrationRequestHandler {
+  private var callback: OneginiBrowserRegistrationCallback? = null
+  private var eventEmitter = RegistrationEventEmitter()
+  /**
+   * Finish registration action with result from web browser
+   */
+  fun handleRegistrationCallback(uri: Uri?): Boolean {
+    callback?.let {
+      callback?.handleRegistrationCallback(uri)
```

Comment:
`?.let` safely unwraps nullable variable, so inside the brackets you're sure that `callback` is non-null. So you should do one of below 
```
callback?.let {
      it.handleRegistrationCallback(uri)
      callback = null
      return true
    }
```
or
```
callback?.let { registrationCallback ->
      registrationCallback.handleRegistrationCallback(uri)
      callback = null
      return true
    }
```
Both approaches are completely the same, in you only **name** it.